packages/server/src/notification/notification.service.ts

```diff
@@ -6,19 +6,26 @@ import { UserModel } from '../profile/user.entity';
 import { DesktopNotifModel } from './desktop-notif.entity';
 import { PhoneNotifModel } from './phone-notif.entity';
 import { TwilioService } from './twilio/twilio.service';
-import twilio from 'twilio';
 
-const phoneResponses = {
-  WRONG_MESSAGE:
-    'Please respond with either YES or NO. Text STOP at any time to stop receiving text messages',
-  COULD_NOT_FIND_NUMBER:
-    'Could not find an Office Hours account with your phone number.',
-  UNREGISTER:
-    "You've unregistered from text notifications for Khoury Office Hours. Feel free to re-register any time through the website",
-  DUPLICATE:
-    "You've already been verified to receive text notifications from Khoury Office Hours!",
-  OK:
-    'Thank you for verifying your number with Khoury Office Hours! You are now signed up for text notifications!',
+export const NotifMsgs = {
```

Comment:
Put all Notification related copy in one spot! Maybe can move this to it's own `notification.constants.ts` file eventually 🤷🏼